packages/server/src/modules/user/mutation/UserLoginWithEmailMutation.ts

```diff
@@ -19,19 +19,18 @@ export default mutationWithClientMutationId({
   mutateAndGetPayload: async ({ email, password }) => {
     const user = await UserModel.findOne({ email: email.toLowerCase() });
 
-    const defaultErrorMessage = 'Invalid password';
 
     if (!user) {
       return {
-        error: defaultErrorMessage,
+        error: 'Email not found',
```

Comment:
You should not tell that email was not found, this is a security problem

Reply:
Right